bootcd/rpms/genesis_scripts/src/root-bash_profile

```diff
@@ -1,14 +1,25 @@
 
-GENESIS_MODE="$(grep -o -e 'GENESIS_MODE=[^ ]*' /proc/cmdline | cut -c14-)"
-
-# setup the ruby environment we need
-source /etc/profile.d/rvm.sh
-rvm list
-ruby --version
-
-if [[ ! -z $GENESIS_MODE ]] && [ ! -e "/var/run/bootloader-autorun.lock" ]
-  then
-    touch /var/run/bootloader-autorun.lock
-    # disable task execution prompting when doing initial target run
-    GENESIS_PROMPT_TIMEOUT=0  /usr/bin/genesis-bootloader 2>&1 | tee /var/log/genesis-bootloader.log
+# Genesis modifications
+
+# Get our configuration
+source /etc/sysconfig/genesis
+
+# Try for two seconds to find a pidfile
+for i in {1..20}
```

Comment:
I'm pushing back on this one. I looked in `/etc/init.d/functions` and found nothing that does what I want which may not be totally obvious:
The pid file is used to wait for the bootloader to have started but I don't care if it's currently running.
The init script first starts logging and then drops a pid file which means if I wait for the pid file I am guaranteed that logging to the log file has begun and the tail should work correctly.

If you know of a particular function that waits a reasonable amount of time for a file to exist, I'm happy to use it, but I didn't find any obvious candidates.